sys/include/net/ng_pkt.h

```diff
@@ -34,10 +34,7 @@ extern "C" {
  *          snip == (packet header || packet payload)) of a network packet
```

Comment:
Ah, I overlooked the second closing one. But then I don't understand what the "or parts".

I would prefer something like:
```C
/**
  * @brief   Type to represent parts of a packet, called snips.
  *          A snip is either a header or the payload of a network packet.
```